fcl/caf/set_caf_calconst.fcl

```diff
@@ -0,0 +1,6 @@
+physics.producers.pandoraCaloGausCryoE.CaloAlg.CalAreaConstants: @local::this_cal_constants
+physics.producers.pandoraCaloGausCryoW.CaloAlg.CalAreaConstants: @local::this_cal_constants
+physics.producers.vertexChargeCryoE.CaloAlg.CalAreaConstants: @local::this_cal_constants
+physics.producers.vertexChargeCryoW.CaloAlg.CalAreaConstants: @local::this_cal_constants
+physics.producers.vertexStubCryoE.CaloAlg.CalAreaConstants: @local::this_cal_constants
+physics.producers.vertexStubCryoW.CaloAlg.CalAreaConstants: @local::this_cal_constants
```

Comment:
Several shower algs also set the Calorimetry Alg to use and will need to have the cal constants changed.

See e.g. https://github.com/SBNSoftware/icaruscode/blob/feature/gputnam-MCP-configs/fcl/configurations/showerfindermodules_icarus.fcl#L33

specifically looks like tools [3], [5], and [7] -- and these are logged in physics.producers as SBNShowerGausCryoE and SBNShowerGausCryoW

Reply:
Fixed.